lang/py/build.sh

```diff
@@ -84,7 +84,10 @@ main() {
   for target; do
     case "$target" in
       clean) clean;;
-      dist) dist;;
+      dist)
+        dist
+        doc
```

Comment:
It looks to me that the Python SDK distribution does not include its documentation. Is this normal in the Python world ?

Reply:
I don't know if it's normal or not but the web site seems to be intended to include the doc.
https://github.com/apache/avro/blob/master/doc/content/en/docs/%2B%2Bversion%2B%2B/api-py.md?plain=1

Also, I can't find the doc accessible from the PyPi.
https://pypi.org/project/avro/

cc: @kojiromike

Reply:
The Python implementation never published api docs before. I added the sphinx build basics, and modeled it after other implementation docs builds. I know it builds locally, but I don't know if it really works in the expected way yet.

As for what's common in Python, most open source Python projects IME publish their docs to readthedocs/rtfd.io. But I'd consider it a step in the right direction if this implementation published its api docs at all.